miden-crypto-fuzz/fuzz_targets/smt.rs

```diff
@@ -1,12 +1,12 @@
 #![no_main]
 
 use libfuzzer_sys::fuzz_target;
-use miden_crypto::{merkle::Smt, hash::rpo::RpoDigest, Word, Felt, ONE};
+use miden_crypto::{Felt, ONE, Word, hash::rpo::Word, merkle::Smt};
```

Comment:
It's a bit confusing here that we are importing `Word` from both the root and `hash::rpo` modules. I'm curious why lints don't complain about this.

Reply:
Fixed